sys-cluster/charliecloud/charliecloud-0.18.ebuild

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit autotools eutils python-single-r1
```

Comment:
This has grown a bit outdated and there's an eclass called `optfeature` now to use instead of `eutils`.

Reply:
Thanks! 
Is there a mailing list to stay informed of new `eclass` such as this one (I missed that on `gentoo-dev-announce`)?

Reply:
I think it's just gentoo-dev. It's not the worst idea that final eclass changes could be summarised to gentoo-dev-announce though.

Reply:
Ah, ok! Indeed, I am only on `gentoo-dev-announce` since as a proxy-maintainer, the full`gentoo-dev` list is a bit much (would be fine as RSS feed or such, but not for my mailbox which is currently eaten by other projects). If a summary of new eclasses / change of recommendations could be copied to `gentoo-dev-announce`, I believe many would appreciate this :-).

Reply:
I'm wondering if a mail filter could work for now, at least? This is what I do to separate mailing lists into their own folders.